### DISCUSSION TITLE
remote option on connect

### DISCUSSION THREAD

**Discussion by lotfire** [Pomax/msfs-simconnect-api-wrapper#16] [Q&A]
Hello @Pomax ,

first of all thanks a lot for this wrapper ! 

I'm trying to connect remotely by adding the option `const options = { remote: { host: 'localhost', port: 5111 } };`

```
from EvenAR repo but doesn't seem to work.
Here's my code:
`(async function tryConnect() {
  api.connect({
    retries: Infinity,
    retryInterval: 5,
    remote: { host: 'http://192.168.1.112', port: 5111 },
   ...
  });
})();
```

cheers

**Comment by lotfire** [comment]
OK found my mistake :)
Just add a SimConnect.cfg at the root of the node project

```
[SimConnect]
Protocol=IPv4
Address=xxx.xxx.xxx.xxx
Port=xxxx
```

**Comment by Pomax** [comment]
It'd be nice if the code had a way to pass that information directly without needing the `node-simconnect` config file though, let's mark this as a feature request.

**Comment by Pomax** [comment]
filed as https://github.com/Pomax/msfs-simconnect-api-wrapper/issues/17
